src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala

```diff
@@ -0,0 +1,60 @@
+package com.fortysevendeg.android.sqlite
+
+import java.io.{ByteArrayInputStream, InputStream, OutputStream}
+import java.sql.{SQLException, Blob, SQLFeatureNotSupportedException}
+
+class SQLDroidBlob(byteArray: Array[Byte]) extends Blob {
+
+  private[this] val arrayLength = Option(byteArray) map (_.length) getOrElse 0
+
+  def getBytes(pos: Long, length: Int): Array[Byte] = {
+    val newPos = pos - 1
```

Comment:
IMO is not developer friendly, we usually use 0 as initial position, makes sense? Maybe I'm missing something